desktop-widgets/locationinformation.cpp

```diff
@@ -521,7 +521,7 @@ void DiveLocationLineEdit::itemActivated(const QModelIndex &index)
 		qDebug() << "Setting a Existing dive site";
 	if (view->isVisible())
 		view->hide();
-	emit diveSiteSelected(currUuid);
+	emit diveSiteSelected();
```

Comment:
this comment is for the commit message, not for this code... not sure how GitHub will present this comment, so this is about commit https://github.com/Subsurface-divelog/subsurface/pull/1818/commits/3fb3b11f0c0e2f7ad4d953e3f6d0b7041edb1e27 
The commit message says "... receiving slot would not use that signal" - I think it should say "... would not use that argument".

Reply:
true - fixed (locally).